commands/prop.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../lib/package'
+
+class Command
+  def self.prop(property)
+    case property
+    when 'arch_flags_override'
+      puts "Use the 'arch_flags_override' property to override architecture specific flags."
+    when 'conflicts_ok'
+      puts "The 'conflicts_ok' property bypasses checks for other package file conflicts."
+    when 'git_clone_deep'
+      puts "The 'git_clone_deep' property clones the repository without a depth value."
+      puts "Applicable only when 'source_url' is a git repository."
+    when 'git_fetchtags'
+      puts "The 'git_fetchtags' property gets the repository tags."
+      puts "Applicable only when 'source_url' is a git repository."
+    when 'gnome'
+      puts "Use the 'gnome' property to include options for gnome specific packages."
+    when 'is_fake'
+      puts "Use the 'is_fake' property for packages that simply depend on other packages."
+    when 'is_musl'
+      puts "Use the 'is_musl' property for musl specific packages."
+    when 'is_static'
+      puts "Use the 'is_static' property for packages which do not require shared dependencies."
+    when 'no_compile_needed'
+      puts "Use the 'no_compile_needed' property for packages that do not require pre-built binaries."
+    when 'no_compress'
+      puts "Use the 'no_compress' property for packages that do not need compressed files."
+    when 'no_env_options'
+      puts "Use the 'no_env_options' property for packages that do not require"
+      puts 'environment options or to override the default options.'
+    when 'no_git_submodules'
+      puts "Use the 'no_git_submodules' property for repositories without git submodules."
+      puts "Applicable only when 'source_url' is a git repository."
```

Comment:
Put this below `when 'no_fhs'`?  It would be nice to keep all the properties in alphabetical order to prevent potential duplicate lines.

Reply:
Got the alphabetical order wrong :sweat_smile: